### PR TITLE
Supports outputting trade activity to csv file.

### PR DESCRIPTION
**Pull request by ShadoFlameX**
_Depends on https://github.com/bankroll-py/bankroll-model/pull/7._

### CODE REVIEW

File: bankroll/interface/__main__.py [hunk 1]
```diff
@@ -155,6 +160,12 @@ def symbolTimeline(accounts: AccountAggregator, args: Namespace) -> None:
 activityParser = subparsers.add_parser(
     "activity", help="Operations upon imported portfolio activity"
 )
+activityParser.add_argument(
+    "--output-csv",
+    metavar='out-file',
+    type=FileType('w'),
```
Comment (jspahrsummers) [inline bankroll/interface/__main__.py]: I think we should let Pandas open this file for writing. A path can be provided to `to_csv`, so we should be able to just remove this.

File: bankroll/interface/__main__.py [hunk 2]
```diff
@@ -155,6 +160,12 @@ def symbolTimeline(accounts: AccountAggregator, args: Namespace) -> None:
 activityParser = subparsers.add_parser(
     "activity", help="Operations upon imported portfolio activity"
 )
+activityParser.add_argument(
+    "--output-csv",
```
Comment (jspahrsummers) [inline bankroll/interface/__main__.py]: Should we add `-o` as a shorthand for this?

File: bankroll/interface/__main__.py [hunk 3]
```diff
@@ -113,7 +114,7 @@ def printPositions(accounts: AccountAggregator, args: Namespace) -> None:
 
 def printActivity(accounts: AccountAggregator, args: Namespace) -> None:
     if args.output_csv:
-        df = ModelConverter.dataframe(list(accounts.activity())).sort_values(by=['Date'])
+        df = dataframeForModelObjects(list(accounts.activity())).sort_values(by=['Date'])
```
Comment (jspahrsummers) [inline bankroll/interface/__main__.py]: With the change to `Iterable` that I suggested in https://github.com/bankroll-py/bankroll-model/pull/7, this won't need a conversion to `list()` (which is good from a memory perspective!)

File: bankroll/interface/__main__.py (hunk 2 above)
Comment (ShadoFlameX) [reply bankroll/interface/__main__.py]: ✅

File: bankroll/interface/__main__.py (hunk 1 above)
Comment (ShadoFlameX) [reply bankroll/interface/__main__.py]: ✅

File: bankroll/interface/__main__.py (hunk 3 above)
Comment (ShadoFlameX) [reply bankroll/interface/__main__.py]: We still need a list due to fetching the headers and functions for each column separately.

**Pull request merged**